packages/components/src/search-control/types.ts

```diff
@@ -40,6 +40,8 @@ export type SearchControlProps = Pick< InputControlProps, 'help' | 'value' > & {
 	 *
 	 * Use this if you want the button to trigger your own logic to close the search field entirely,
 	 * rather than just clearing the input value.
+	 *
+	 * @deprecated
```

Comment:
Let's also add a `@ignore` tag, which will suppress this from the Storybook props table.

```suggestion
	 *
	 * @deprecated
	 * @ignore
```